pom.xml

```diff
@@ -94,7 +94,7 @@
                 <plugin>
                     <groupId>org.apache.maven.plugins</groupId>
                     <artifactId>maven-compiler-plugin</artifactId>
-                    <version>3.8.1</version>
+                    <version>3.2</version>
```

Comment:
Why downgrade?

Reply:
Using the same version used in java-sqs-extended-client.

Reply:
It is not necessary to have the same version, please revert.

Reply:
reverted to v3.8.1